Src/DependencyCollector/DependencyCollector/DependencyCollector.csproj

```diff
@@ -29,6 +29,10 @@
     <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.9.2">
     <PrivateAssets>All</PrivateAssets>
     </PackageReference>
+  </ItemGroup>
+
+  <ItemGroup>
+    <!--Build Infrastructure-->
```

Comment:
Release + Windows_NT condition would have always been met right? wonder why this caused issues.

Reply:
Yeah i'm still confused why this worked for every other project except Perf.
This is part of what broke the builds for Dmitry this week so I'm making this change everywhere.